src/App.tsx

```diff
@@ -15,36 +16,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NULL = 'null',
+  SORT_BY_ALPHABET = 'alphabetSort',
+  SORT_BY_LENGTH = 'goodsLength',
+}
+
+interface SortReverse {
+  sortField: SortType;
+  isReversed: boolean;
+}
+
+function sortGoods(
+  goods: string[],
+  { sortField, isReversed }: SortReverse,
+): string[] {
+  const newSortGoogs = [...goods];
```

Comment:
There is a typo in the variable name `newSortGoogs`. It should be something like `newSortedGoods` to better reflect its purpose and avoid confusion.